pythonFeed.sh

```diff
@@ -24,6 +25,8 @@ processLastVersion() {
     directoryCheck "$majorMinor" "$SEARCH_TERM"
     if [[ $(eval echo $?) == 0 ]]; then
         echo "$LATEST_VERSION is valid"
+        echo "generating cimg-python for version: $LATEST_VERSION"
+        ./shared/release.sh "$LATEST_VERSION"
```

Comment:
wouldn't this just create a new PR for every version?

Reply:
Thats the goal, because it seemed to fail with generating one PR for all 3 https://app.circleci.com/pipelines/github/CircleCI-Public/cimg-python/471/workflows/42535d25-066e-43cf-aec1-eac5545fa6d3/jobs/516

Reply:
it failed to generate one PR for all three because the entire string was evaluated, rather than by each version within in string

Reply:
@JalexChen turns out I literally just needed to take a 'v' on in front of my versions, it should work fine with this new minimal change